storage/inkless/src/main/resources/db/migration/V7__Find_batches_function.sql

```diff
@@ -0,0 +1,120 @@
+-- Copyright (c) 2025 Aiven, Helsinki, Finland. https://aiven.io/
+
+CREATE TYPE find_batches_request_v1 AS (
+    topic_id topic_id_t,
+    partition partition_t,
+    starting_offset BIGINT,
+    max_partition_fetch_bytes INT
```

Comment:
should we move this to `find_batches_v1` as input parameter as it's always the same on the same request?